README.md

```diff
@@ -1,6 +1,19 @@
-# the pal boilerplate
-
-A friendly, proven starting place for your next hapi plugin or deployment
+<div align="center">
+  <br>
+  <a href="https://hapipal.com" target="_blank"><img src="https://imgur.com/shaShr0.png" alt="hapipal" width="200"></a>
+  <br>
+  <h1 align="center">
+  hapipal boilerplate
+  <br>
+  <a
+  target="_blank"  href='https://join.slack.com/t/hapihour/shared_invite/enQtMjM5Njk1NDgzNTY5LThmODkxODIzOTg5NjJjODFiYjcxZDMxMTAyMzBkZDk3MWY4MTFlNDAyMTU3MmUwMmM0Y2UwMjU3YjAwYjRkN2E'>
```

Comment:
hmmm so it looks like the target attribute isn't supported in Github markdown: https://stackoverflow.com/questions/41915571/open-link-in-new-tab-with-github-markdown-using-target-blank (verified in "testing" / clicking on these images :trollface: )

guessing for security reasons?

also, looks like Github automatically adds `rel=nofollow` to all a tags. Also not totally sure why
for your health!